web-api/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.ts

```diff
@@ -155,7 +155,7 @@ export const fileAndServeCourtIssuedDocument = async (
           {
             ...omit(docketEntryToServe, 'filedBy'),
             attachments: form.attachments,
-            date: form.date,
+            date: form.date || createISODateString(form.dueDate),
```

Comment:
Are we sure that `date` is always meant to be a due date? This is changing persistence, which makes me nervous given what this ticket is doing.